### ISSUE TITLE
This should be a non-profit organisation

### ISSUE THREAD

**Issue by michaelbarton** [bioboxes/rfc#39]
I believe this organisation would benefit from being incorporated as a
non-profit entity. This would allow us to accept donations and use these funds
where appropriate such as sending members to conferences, running a conference,
or giving members grants to perform work that would benefit the project. There
may also be 'soft' benefits such a lending legitimacy to the project through
indication that we intend this to be long-running and community-driven.

An existing precedent in bioinformatics are the Bio* library projects such as
BioPerl. The Bio* libraries are under the [Open Bioinformatics Foundation
(OBF)][obf] which in turn is under [Software in the Public Interest, Inc
(SPI)][spi].

The [Python Software Foundation][psf] is incorporated as a non-profit and
manages itself. I am a little weary of this approach as this may require
additional bureaucratic work to manage legal aspects and this may detract from
the core mission of defining and managing community standards.

Another approach is joining the Apache Software Foundation through their
[incubator][asf]. Apache has a pedigree for running open source projects and
that experience and infrastructure could be useful for this project.

In summary I see at least the following options:

  * Incorporate and manage the non-profit organisation ourselves similar to the
    [Python Software Foundation][psf].
  * Follow the bio* libraries' path and inquire about joining [SPI][spi].
  * Attempt to join the Apache Software Foundaton through their
    [incubator][apache].

Any other suggestions are welcome.

[obf]: http://www.open-bio.org/wiki/Main_Page
[spi]: http://spi-inc.org/
[psf]: https://www.python.org/psf/
[apache]: http://incubator.apache.org/

**Comment by michaelbarton** [comment]
@avilella This is in response to our conversation.

**Comment by chapmanb** [comment]
Michael;
Would you have any interest in joining the Open Bioinformatics Foundation community directly? This is definiately a project of interest to many members of OBF, myself included, and you could benefit from not having to setup the non-profit infrastructure.

We worked a year or so ago on having a policy to add additional affiliated projects but it lost steam a bit without anyone leading it:

https://github.com/OBF/obf-docs/blob/unratified-drafts/Affiliated-Project-Policy.md

The general idea was to allow more folks to be officially involved with OBF but how to do this fairly. This is still not sorted out but you could help revive the discussion. I'm cc'ing @peterjc and @hlapp who are actually in charge of decision making at OBF, but just opening up a conversation.

**Comment by michaelbarton** [comment]
Thanks for your comment Brad. I am unfamiliar with creating non-profit
organisation as I have never proceeded down this path before. I think it could
be useful to join an existing organisation as it would preclude us having to
manage legal aspects. Could you tell us a bit more about how the OBF works in
developing a spec like this? Will we have to follow the OBFs infrastructure for
managing an organisation?

**Comment by peterjc** [comment]
To be clearer, Hilmar (@hlapp) and myself are currently president and secretary of the OBF (http://www.open-bio.org - @OBF on GitHub), and only two of the current board of directors. But thanks Brad (@chapmanb), yes, it might make sense for SIBCI (name change pending, #40) to be part of the OBF. However, note that the OBF currently does not keep specific sub-funds earmarked for each member project. Right now our main outgoings are server costs (AWS), domain names, some travel grants and our conference BOSC (which aims to be revenue neutral).

I have yet another suggestion for you to consider - an ISCB affiliated COSI. You mentioned running a conference as a possible reason for forming a non-profit. We current run BOSC as a SIG (special interest group) satellite meeting at the annual ISMB conference run by the ISCB, where ISCB holds our BOSC float in escrow. They have various options for holding meetings including workshops etc, and the new ISCB COSI scheme (communities of special interest, which we have joined as the OBF including BOSC), allows use of the escrow funds during the year for other meeting activities (not just at the main annual ISMB conference). See http://www.iscb.org/iscb-affiliates-cosis

**Comment by chapmanb** [comment]
Michael;
The goal of OBF is to connect and support researchers working in open source bioinformatics. Beyond that, there isn't a process or formalized requirements for getting work done. You'd basically be able to work on the science/specification part however you see fit. We only want to bring people together to make it easier to build communities.

Peter is much more in-touch with the money/logistics side of things and I'd defer to his thoughts on the best plans for that. Thanks again for all your work on this.

**Comment by avilella** [comment]
Hi Brad,

Thanks for the details. Quick question for you/Peter:

If a company wants to sponsor bioboxes, so that, for example, @mbarton can
attend a conference: can the company send the £500 directly to the bioboxes
account to pay for these expenses?

This is, if bioboxes is part of OBF, can companies do this kind of "spot
sponsoring" easily without much paperwork?

Cheers,

A.

**Comment by peterjc** [comment]
The OBF does not keep specific sub-funds earmarked for each member project, but we do make this kind of sponsorship arrangement for our annual meeting BOSC. In some cases the sponsor gives us money (which can count as a donation which is often advantageous for US taxes), and we reimburse the costs, while in a few cases the sponsor has directly paid (eg catering). We have also provided travel support to our meeting in the past, and are looking into formalising this as an "OBF Travel Fellowship".

Usually the worst of the paperwork has been preparing a suitable invoice to get the agreed sponsorship out of the companies' finance departments.

**Comment by avilella** [comment]
Cool, thanks. What if the company wants to sponsor something specific or
sponsor someone for a specific conference, not just generally give
conference money for BOSC?

**Comment by peterjc** [comment]
To date most of our BOSC associated sponsorship has been earmarked for a particular thing (which we would mention in our acknowledgement slides, emails, etc). For example last year Google paid for video recording, Eagle paid for student prizes, while Curoverse and Harbinger Partners specifically supported our CodeFest. See http://www.open-bio.org/wiki/BOSC_2014 and http://www.open-bio.org/wiki/Codefest_2014

**Comment by avilella** [comment]
Can it be for a specific purpose and a specific person within the
organization?

For example: £500 for conference attendance for Michael Barton from
SponsorXYZ

**Comment by peterjc** [comment]
I don't see any reason not too, @hlapp would that need @OBF board discussion and approval?

P.S. I like the new name *bioboxes*, it would look right at home amongst the other OBF Bio* projects ;)

**Comment by avilella** [comment]
That would be great! We will await for Hilmar's confirmation :-)

**Comment by hlapp** [comment]
The only constraint is that the earmarked purpose has to be one that is compatible under 501(c)3 rules. So not every sponsorship can be paid out for any purpose. But travel support if the travel is evidently in furthering the project's goals and mission should be fine. At the end of the day, the ruling is in the hands of the fiscal sponsor, i.e., SPI in our case, because they are the ones who are responsible to the IRS.

**Comment by michaelbarton** [comment]
Thank you all for taking the time to comment on this issue, it's very much
appreciated and helps us understand the process. I am certainly very unfamiliar
with how to evaluate this decision as I have never had to tread this path
previously.

If the circumstances for bioboxes changed in the future would be it relatively
easy to diverge and possibly create a separate organisation or join another? If
so then this means this discussion may be less weighty since it is not an
irreversible decision.

**Comment by hlapp** [comment]
Not only is the decision reversible, it is also not exclusive. I.e., there is nothing that prevents you from collecting funds outside of OBF or the fiscal sponsor, and you can also choose to have other fiscal sponsors in addition. The only thing to keep in mind is that assets held by the fiscal sponsor can only be transferred to another 501(c)3 approved organization.

**Comment by michaelbarton** [comment]
After our conference call #50, @vilella will take the lead on guiding this
process and making decisions.

**Comment by avilella** [comment]
- https://www.gov.uk/government/publications/what-makes-a-charity-cc4/what-makes-a-charity-cc4#part-3-about-the-descriptions-of-purposes

- Usually, you must register with the Charity Commission if your
  charity is based in England or Wales and has over £5,000 income per
  year.

**Comment by michaelbarton** [comment]
Discussed in meeting #67. @avilella has been looking into creating a non-profit
UK charity. A charity doesn't need to be registered if there is less than
£15,000 in funding, and the donors do not need tax receipts.

@avilella if no further action is required on this issue, can this issue be
closed?

**Comment by michaelbarton** [comment]
@avilella if we wanted to pay for travel expenses as a non-profit rather than
through onto.it, can this be done?

**Comment by avilella** [comment]
I think the procedure would be for members of bioboxes to pay up-front and
get directly reimbursed to their bank accounts. I'll ask if that is ok on
my side...

**Comment by michaelbarton** [comment]
This issue is currently moot. There are no organisations who will sponser bioboxes development. If this situation changes in future we will reopen this ticket.
